stable/insights-agent/requirements.lock

```diff
@@ -1,12 +1,12 @@
 dependencies:
 - name: goldilocks
   repository: https://charts.fairwinds.com/stable
-  version: 3.2.7
+  version: 4.0.1
 - name: prometheus
   repository: https://prometheus-community.github.io/helm-charts
-  version: 14.7.1
+  version: 14.11.0
 - name: insights-admission
   repository: https://charts.fairwinds.com/stable
   version: 0.4.0
```

Comment:
I'm not familiar with the "generated" aspect of this, but I wonder whether its version should change upstream, since it was re-generated and has a different hash? Feel free to ignore this comment if it is something I have yet to understand.

Reply:
I think you're on the right track - it's changed in requirements.yaml, and then this file gets updated when you run `helm dependency update`

Reply:
It looks like the `admission` version remains at 0.4.0, yet the `digest` and `generated` in release.lock changed?